web/src/components/FilingsCalendar.tsx

```diff
@@ -84,6 +85,17 @@ export const FilingsCalendar = (props: Props): ReactElement => {
     );
   };
 
+  const handleCalendarOnclick = () => {
+    if (userData)
```

Comment:
we prefer guard clauses (as in, `if (!userData) return` instead of nesting inside of an if-statement. The reason is that it's easier to read code (especially the core code of the function) that has as little nesting as possible, and that is as close to left-aligned with the function margin as possible.